.github/workflows/experimental.yml

```
@@ -0,0 +1,102 @@
+on:
+  workflow_call:
+    inputs:
+      jdk_version:
+        description: 'JDK version'
+        required: true
+        type: string
+      pull_request_repo:
+        description: 'Repository to check out'
+        required: true
+        type: string
+        default: 'jbosstm/narayana'
```

actionlint (1.7.12)

[error] input "pull_request_repo" of workflow_call event has the default value "jbosstm/narayana", but it is also required. if an input is marked as required, its default value will never be used

(events)